src/pypgstac/pypgstac/db.py

```diff
@@ -86,7 +87,7 @@ def get_pool(self) -> ConnectionPool:
                 num_workers=settings.db_num_workers,
                 kwargs={
                     "options": "-c search_path=pgstac,public"
-                    " -c application_name=pypgstac"
+                    " -c application_name=pypgstac",
```

Comment:
@bitner could you please make `PgstacDB` more robust by not using command-line options? As @captaincoordinates [pointed out](https://github.com/stac-utils/pgstac/issues/156) `PgstacDB`  cannot work properly with Amazon RDS Proxy in its current state. The possible solution would be to remove that, and then once the connection is established execute:
```
self.connection.execute(
    "set search_path = pgstac,public; set application_name=pypgstac"
)
```

Reply:
@drnextgis @captaincoordinates, I've made a change for this in the latest commit. Can you verify that this works with RDS Proxy?

Reply:
@bitner thanks for adding this change, unfortunately it will be a little while before I can test this on RDS Proxy - I'm currently using pypgstac via stac-fastapi and it doesn't support 0.7.0. I'll have to create a new project and some new AWS infrastructure for it to use before I can verify.